test/dfusionMultiSafes.js

```diff
@@ -21,11 +21,10 @@ const {
   buildTransferFundsToMaster,
   buildWithdrawAndTransferFundsToMaster,
   isOnlySafeOwner,
-  max128,
   maxU32,
 } = require("../scripts/utils/trading_strategy_helpers")(web3, artifacts)
 const { waitForNSeconds, execTransaction } = require("../scripts/utils/internals")(web3, artifacts)
-const { checkCorrectnessOfDeposits } = require("../scripts/utils/price-utils")(web3, artifacts)
```

Comment:
I see now, that it was moved from `helpers` to `utils`. If we are going to move this then we should also be moving `maxU32` along with it. Otherwise I would say that the change is not justified.